crates/uv/tests/it/python_find.rs

```diff
@@ -663,3 +664,82 @@ fn python_find_venv_invalid() {
     ----- stderr -----
     "###);
 }
+
+/// See: <https://github.com/astral-sh/uv/issues/11825>
+#[test]
+#[cfg(unix)]
+fn python_required_python_major_minor() {
+    let context: TestContext = TestContext::new_with_versions(&["3.11", "3.12"]);
+
+    // Find the Python 3.11 executable.
+    let path =
+        String::from_utf8(context.python_find().arg("3.11").output().unwrap().stdout).unwrap();
+    let path = Path::new(path.trim());
```

Comment:
You can use `context.python_versions.first().unwrap().1`

Reply:
Great, thank you!